pkg/v1/tkg/test/tkgctl/tkgs_cc/tkgs_cc_workload_cluster_test.go

```diff
@@ -62,6 +62,11 @@ var _ = Describe("TKGS ClusterClass based workload cluster tests", func() {
 		It("should successfully create a cluster", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should successfully upgrade a cluster", func() {
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: seems useless, same as below

Reply:
yes it does. need to fix.

Reply:
yes it would seem so.. lets talk about it.